tests/checks/test_cloudbuilder.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import absolute_import
+
+import json
+from textwrap import dedent
+
+import pytest
+import responses
+
+from freight import checks
+from freight.exceptions import CheckFailed, CheckPending
+from freight.testutils import TestCase
+
+
+class CloudbuilderCheckBase(TestCase):
+    def setUp(self):
+        self.check = checks.get("cloudbuilder")
+        self.user = self.create_user()
+        self.repo = self.create_repo()
+        self.app = self.create_app(repository=self.repo)
+        self.test_project = "mycoolproject"
+        self.test_sha = "0987654321"
+        self.test_token = "mysuperfaketoken"
+
+
+class CloudbuilderContextCheckTest(CloudbuilderCheckBase):
+    @responses.activate
+    def test_build_success(self):
+        id = "successful_build_id"
+        body = json.dumps({
+            "builds": [{
+                "id": "{}".format(id),
```

Comment:
You're formatting a blank string with a string. You can shorten this to `"id": build_id`.